lib/VolSurface/Calibration/Equities.pm

```diff
@@ -0,0 +1,637 @@
+package VolSurface::Calibration::Equities;
```

Comment:
Where do we use this module? Or rather how often? It seems to do quite complex maths. Is it worth to covert that to XS having the core loops in C?